src/common/entity/get_states.ts

```diff
@@ -122,13 +122,14 @@ const FIXED_DOMAIN_ATTRIBUTE_STATES = {
   },
   climate: {
     hvac_action: [
```

Comment:
This list is shown in this order and thus not alphabetically sorted for a reason.

Please revert the sorting made here.